crates/core/tedge_mapper/src/aws/mapper.rs

```diff
@@ -30,6 +33,30 @@ impl TEdgeComponent for AwsMapper {
     ) -> Result<(), anyhow::Error> {
         let (mut runtime, mut mqtt_actor) =
             start_basic_actors(self.session_name(), &tedge_config).await?;
+        if tedge_config.mqtt.bridge.built_in {
+            let device_id = tedge_config.device.id.try_read(&tedge_config)?;
+            let rules = built_in_bridge_rules(device_id)?;
+
+            let mut cloud_config = rumqttc::MqttOptions::new(
+                tedge_config.device.id.try_read(&tedge_config)?,
+                tedge_config.aws.url.or_config_not_set()?.to_string(),
+                8883,
```

Comment:
Something is wrong here. I would expect call to `host()` and `port()` on `tedge_config.aws.url`.

Reply:
Again, this is a `ConnectUrl`